doc/tutorials/plugins.md

```diff
@@ -322,6 +323,20 @@ The `elektraPluginOpen` and `elektraPluginClose` functions are not commonly used
 reviewing. `elektraPluginOpen` function runs before `elektraPluginGet` and is useful to do initialization if necessary for the plug-in. On the other
 hand `elektraPluginClose` is run after other functions of the plug-in and can be useful for freeing up resources.
 
+### elektraPluginCheckConf ###
+
+The `elektraPluginCheckConf` function may be used for validation of the plugin configuration during mount time. The signature of the function is:
+
+	int elektraLineCheckConfig (Key * errorKey, KeySet * conf)
+
+The configuration of the plugin is provided as `conf`. The function may report errors using the `errorKey` and the return value.
+
+The following convention was established for the return value of `elektraPluginCheckConf`:
+
+- 0: config was not changed (was ok)
+- 1: config is changed (now ok)
+- -1: config not ok, could not fix it
```

Comment:
The text was not supposed to be used 1:1, it is not even always a sentence ;)

It also does not establish that iff the function returns -1 an error has to be set.

Also please give an example.